rpc/src/v1/helpers/errors.rs

```diff
@@ -100,6 +100,14 @@ pub fn request_rejected_limit() -> Error {
 	}
 }
 
+pub fn request_rejected_param_limit() -> Error {
+	Error {
+		code: ErrorCode::ServerError(codes::REQUEST_REJECTED_LIMIT),
+		message: "Request has been rejected because requested data size exceeds limit.".into(),
```

Comment:
How a user can find out the value of the limit?

Reply:
I'll put it in the error message.